shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -823,3 +823,28 @@ async fn events_keyspace(#[case] driver: CassandraDriver) {
 
     shotover.shutdown_and_then_consume_events(&[]).await;
 }
+
+#[rstest]
+#[case::cdrs(CdrsTokio)]
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_protocol_v5(#[case] driver: CassandraDriver) {
+    let _docker_compose =
+        DockerCompose::new("example-configs/cassandra-passthrough/docker-compose.yaml");
+
+    let shotover = ShotoverProcessBuilder::new_with_topology(
+        "example-configs/cassandra-passthrough/topology.yaml",
```

Comment:
we should use topology-encode.yaml here to get better coverage of our v5 support.